SR2025/2024-11-13-check-in.md

```diff
@@ -0,0 +1,26 @@
+---
+to: SR2025 teams
+subject: SR2025 November Check-in
```

Comment:
```suggestion
subject: Student Robotics November Check-in
```
If this is the subject that we're using on the email itself

Reply:
It is, I've been trying to keep both SR/Student Robotics and the year in the subject line so that the emails are identifiable among other emails and distinguishable from year to year.

Reply:
That's handy to have. Think we should write Student Robotics in full in the subject of emails, particularly on an email checking up on teams that haven't done anything yet

Reply:
8b74f86db207727f151262c85b92942b085244e2